gsplat/project_gaussians.py

```diff
@@ -15,7 +15,7 @@ def project_gaussians(
     glob_scale: float,
     quats: Float[Tensor, "*batch 4"],
     viewmat: Float[Tensor, "4 4"],
-    projmat: Float[Tensor, "4 4"],
+    projmat: Optional[Float[Tensor, "4 4"]],
```

Comment:
I get that this is good for backward compatibility, but if we're going to phase out the projection matrix, we should ultimately drop this argument.